File: service/src/main/java/hu/bsstudio/robonaut/race/speed/timer/SpeedTimerService.java
package hu.bsstudio.robonaut.race.speed.timer;

import hu.bsstudio.robonaut.race.speed.timer.model.SpeedTimer;
import reactor.core.publisher.Mono;

public interface SpeedTimerService {
    Mono<SpeedTimer> updateTimer(final SpeedTimer speedTimer);
}

File: server/service/src/main/kotlin/hu/bsstudio/robonaut/race/speed/timer/SpeedTimerService.kt
package hu.bsstudio.robonaut.race.speed.timer

import hu.bsstudio.robonaut.race.speed.timer.model.SpeedTimer
import reactor.core.publisher.Mono

interface SpeedTimerService {
    fun updateTimer(speedTimer: SpeedTimer): Mono<SpeedTimer>
}
